tests/unit/amo/pages/TestAddon.js

```diff
@@ -1190,53 +1190,20 @@ describe(__filename, () => {
   });
 
   it('renders recommendations for an extension', () => {
-    const fakeConfig = getFakeConfig({
-      enableFeatureAddonRecommendations: true,
-    });
+    const fakeConfig = getFakeConfig();
     const addon = createInternalAddon(fakeAddon);
     const root = shallowRender({ addon, config: fakeConfig });
     expect(root.find(AddonRecommendations)).toHaveLength(1);
     expect(root.find(AddonRecommendations)).toHaveProp('addon', addon);
   });
 
   it('renders recommendations for an extension with no loaded add-on', () => {
-    const fakeConfig = getFakeConfig({
-      enableFeatureAddonRecommendations: true,
-    });
+    const fakeConfig = getFakeConfig();
     const root = shallowRender({ addon: null, config: fakeConfig });
```

Comment:
You can remove `fakeConfig` and remove `config: fakeConfig` in `shallowRender()`, it is not needed anymore.

Reply:
Oh, okay, working on it right away!